index.css

```diff
@@ -0,0 +1,3 @@
+.header{
+    color: brown;
+}
```

Comment:
- [ ] Please, fix the linter issue
<img width="844" alt="Screen Shot 2022-01-06 at 11 06 48" src="https://user-images.githubusercontent.com/49064106/148343012-6e0554a1-a992-478a-bc41-618acd28c4a6.png">